machiavelli-cards/src/main/resources/application.properties

```diff
@@ -1 +1,18 @@
+# DB config
+spring.datasource.url=jdbc:h2:mem:machiavelli-dev-db
```

Comment:
move to config java class and use Hikari

Reply:
Is it possible later to change configurations during deployment according to evniroment type?

Reply:
yes - we still extract the values from the application properties - but we'll use PersistenceConfig with HikariConfig to manage our db connection